service/src/test/java/org/cbioportal/service/impl/MutationServiceImplTest.java

```diff
@@ -176,17 +176,17 @@ public void fetchMetaMutationsInMolecularProfile() throws Exception {
         Mockito.when(molecularProfileService.getMolecularProfile(MOLECULAR_PROFILE_ID)).thenReturn(molecularProfile);
 
         MutationMeta expectedMutationMeta = new MutationMeta();
-        Mockito.when(mutationRepository.fetchMetaMutationsInMolecularProfile(MOLECULAR_PROFILE_ID, 
+        Mockito.when(mutationRepository.fetchMetaMutationsInMolecularProfile(MOLECULAR_PROFILE_ID,
             Arrays.asList(SAMPLE_ID1), Arrays.asList(ENTREZ_GENE_ID_1))).thenReturn(expectedMutationMeta);
-        MutationMeta result = mutationService.fetchMetaMutationsInMolecularProfile(MOLECULAR_PROFILE_ID, 
+        MutationMeta result = mutationService.fetchMetaMutationsInMolecularProfile(MOLECULAR_PROFILE_ID,
             Arrays.asList(SAMPLE_ID1), Arrays.asList(ENTREZ_GENE_ID_1));
 
         Assert.assertEquals(expectedMutationMeta, result);
     }
 
     @Test(expected = MolecularProfileNotFoundException.class)
     public void fetchMetaMutationsInMolecularProfileNotFound() throws Exception {
-        
```

Comment:
You need to take a look the IDE you are using, it should understand the settings in the .editorconfig file.  We have `trim_trailing_whitespace = false` to force the IDE won't trim the trailing whitespace which is what happening here and many other places.